packages/editor/src/styles/editor.css

```diff
@@ -243,8 +243,7 @@ ul[data-type="taskList"] li > div > p {
 
 ul[data-type="taskList"] li[data-checked="true"] > div > p {
   color: rgb(var(--color-text-400));
-  text-decoration: line-through;
-  text-decoration-thickness: 2px;
+  transition: color 0.2s ease;
```

Comment:
Are we changing the color of checked items? Isn't the ticked checkbox enough?

Reply:
We are changing the text of checked items. Once something's done, we don't want to pay much attention to it, hence a muted text color. Removed the `line-through` as it was affecting the readability.